roles/backend_setup/tasks/thick_lv_create.yml

```diff
@@ -1,8 +1,9 @@
+---
```

Comment:
@Sheersha-jain can you please remove the dashes, this will cause syntax error.

Reply:
@sac if I am removing then error is coming 

--> Executing Yamllint on files found in /root/mywork/gluster-ansible-infra/roles/backend_setup/...
    /root/mywork/gluster-ansible-infra/roles/backend_setup/tasks/thick_lv_create.yml
      2:1       warning  missing document start "---"  (document-start)

Reply:
@Sheersha-jain that was my mistake. Please ignore the comment.

Reply:
The dashes are ok.